tortoise/fields/relational.py

```diff
@@ -374,6 +374,7 @@ def describe(self, serializable: bool) -> dict:
         desc["backward_key"] = self.backward_key
         desc["through"] = self.through
         desc["on_delete"] = str(self.on_delete)
+        desc["create_unique_index"] = self.create_unique_index
```

Comment:
Oh, why is it called `create_unique_index` if we use just `unique` for other fields? It feels not clean to have different names for the same configuration options.

Before we propagate this naming to aerich, I suggest we rename `create_unique_index` to `unique` but keep `create_unique_index` deprecated for backwards compatibility.

Reply:
Agree. Will do it later.